docs/utilities/animations.md

```diff
@@ -1560,14 +1560,258 @@ Due to a bug in Safari versions 9-11, stepping through animations via `progressS
 
 ## Interfaces
 
+### AnimationBuilder
+
+```tsx
+type AnimationBuilder = (baseEl: any, opts?: any) => Animation;
+```
+
+:::note
+
+`opts` are additional options that are specific to the custom animation. For example, the sheet modal enter animation includes information for the current breakpoint.
+
+:::
+
+### Animation
+
+```tsx
+interface Animation {
+  parentAnimation?: Animation;
+  elements: HTMLElement[];
+  childAnimations: Animation[];
+  id: string | undefined;
+
+  /**
+   * Play the animation
+   *
+   * If the `sync` options is `true`, the animation will play synchronously. This
```

Comment:
Isn't this the same information that is listed in the properties/methods tables below?